Source/Core/DolphinQt2/Config/Graphics/GraphicsWindow.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QDialog>
+
+class QTabWidget;
+class QDialogButtonBox;
+
+namespace X11Utils
+{
+class XRRConfiguration;
+}
+
+class GraphicsWindow final : public QDialog
+{
+  Q_OBJECT
+public:
+  explicit GraphicsWindow(QWidget* parent, X11Utils::XRRConfiguration* xrr_config);
```

Comment:
- The `QWidget* parent` argument should come at the end of the argument list, to match Qt.
- Does `XRRConfiguration` need to be passed in to this widget? Can it be created inside, or have the relevant information passed in directly?